samples/ldscripts/microbit.ld

```diff
@@ -1,40 +1,40 @@
-/* Linker script to configure memory regions. 
- * Need modifying for a specific board. 
+/* Linker script to configure memory regions.
+ * Need modifying for a specific board.
  *   FLASH.ORIGIN: starting address of flash
  *   FLASH.LENGTH: length of flash
  *   RAM.ORIGIN: starting address of RAM bank 0
  *   RAM.LENGTH: length of RAM bank 0
  */
 MEMORY
 {
-  FLASH (rx) : ORIGIN = 0x00000000, LENGTH = 0x40000
-  RAM (rwx)  : ORIGIN = 0x20000000, LENGTH = 0x4000
+    FLASH (rx) : ORIGIN = 0x00000000, LENGTH = 0x40000
```

Comment:
picolibc.ld seems to provide a way to override the addressed and sizes, e.g.

flash (rx) : ORIGIN = DEFINED(__flash) ? __flash : 0x10000000, LENGTH = DEFINED(__flash_size) ? __flash_size : 0x10000

would it be possible to just use picolibc.ld with correct addresses specified elsewhere?